build/miengine.targets

```diff
@@ -26,6 +26,27 @@
     </Task>
   </UsingTask>
 
+    <!-- Localize the output assembly if requested -->
+  <ItemGroup Condition=" '$(LocalizeOutputAssembly)'=='true' ">
+    <FilesToLocalize Include="$(TargetPath)">
+      <TranslationFile>$(RepositoryRoot)LOC\LCL\{Lang}\$(TargetFileName).lcl</TranslationFile>
```

Comment:
If we care about cross compat if we need to be able to build on Linux in the future, these should match the case of the folders in the repo